bundles/org.openhab.binding.openweathermap/src/main/java/org/openhab/binding/openweathermap/internal/connection/OpenWeatherMapConnection.java

```diff
@@ -379,6 +379,8 @@ private String getResponse(String url) {
                     logger.debug("OpenWeatherMap server responded with status code {}: {}", httpStatus, errorMessage);
                     throw new ConfigurationException(errorMessage);
                 case TOO_MANY_REQUESTS_429:
+                    errorMessage = getErrorMessage(content);
+                    logger.debug("OpenWeatherMap server responded with status code {}: {}", httpStatus, errorMessage);
```

Comment:
This can be reverted, since is falls through (no `break`) - so now logged twice.

Reply:
I’ll add a break as switches without breaks are horrible to read.

Reply:
Changed